code/src/components/TaskList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components/macro'
+
+import tasks from '../reducers/tasks'
+import EmptyStateAnimation from './EmptyStateAnimation'
+
+const TaskContainer = styled.div`
+    overflow: auto;
+    width: 100%;
+`
+
+const SingleTaskContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 4px;
+    margin-bottom: 8px;
+    border-bottom: 1px solid black;
+    word-break: break-word
+`
+
+const Checkbox = styled.input`
+    width: 10%
+    margin: 0;
+`
+
+const Normaltext = styled.p`
+    font-size: 12px;
+    margin: 0;
+`
+
+const DeleteButton = styled.button`
+    width: 13px;
+    height: 13px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 3px 4px 3px 3px;
+    background-color: #E5E5E5;
+    border: 1px solid grey;
+    border-radius: 3px
+`
+
```

Comment:
I know you said you were gonna work with styled component for the first time (i think) this week and it turned out great! ⭐  I also prefer that now, lets find out the advanced stuff too, React for life haha! 😊😊